test/Microsoft.Health.Fhir.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -55,6 +59,265 @@ public async Task WhenGettingSystemHistory_GivenNoType_TheServerShouldReturnTheA
                 Assert.NotEmpty(readResponse.Resource.Entry);
         }
 
+        [Fact]
+        public void WhenGettingSystemHistory_GivenAValueForSince_TheServerShouldReturnOnlyRecordsModifiedAfterSinceValue()
+        {
+            var since = GetStartTimeForHistoryTest();
+            var sinceUriString = HttpUtility.UrlEncode(since.ToString("o"));
+            Thread.Sleep(1000);
+            _createdResource.Resource.Comment = "Changed by E2E test";
+
+            var updatedResource = Client.UpdateAsync<Observation>(_createdResource).GetAwaiter().GetResult();
+
+            Thread.Sleep(5000); // ensure that the server has fully processed the PUT
```

Comment:
Not sure about needing this, should the CosmosDb sessions not handle this type of consistency? (`SessionMessageHandler`) // @johnstairs

Reply:
We use session consistency in the E2E tests and take care of passing around the session token. You should not need this.

Reply:
I have removed or reduced the wait times.